---
layout: page
title: Module 1
permalink: /module1/
---


# Module 1. Foundations of Bioinformatics

Bioinformatics is driving the collection, analysis and interpretation of big data for biomedicine and the biosciences.  This module provides an introduction to practical issues of computer-based handling and interpretation of biomolecular and genomic datasets.  We specifically target bioinformatics software and data resources freely available on the Internet.

<br>

#### Lecture (1-1):	**Introduction to bioinformatics**  
- **Instructor**: 	Dr. Barry Grant  
- **Time**: 		Jan 12 (Tuesday), 2:30 - 4:00 PM  
- **Topics**:  
What are bioinformatics and computational biology?  Major NCBI and EBI bioinformatics resources.  How do we actually do bioinformatics?  Major areas of research and application.  
- **Material**: [Pre class screen cast], [Lecture Slides], [Major Bioinformatics Databases]
- **Homework**: [Readings], [Questions]  

#### Lab (1-1): 	Bioinformatics databases and major online resources  
- **Instructor**: 	Dr. Barry Grant  
- **Time**: 		2:30 – 4:00 PM, Jan 14 (Thursday) or Jan 15, 10:30 - 12:00 PM, (Friday)  
- **Topics**:  
The purpose of this lab session is to introduce a range of core bioinformatics databases and associated online services whilst investigating the molecular basis of several common human disease.  
- **Material**: [Lab worksheet]

<br>

#### Lecture (1-2): **Sequence alignment and database searching** 
- **Instructor**: 	Dr. Barry Grant  
- **Time**: 		Jan 19 (Tuesday), 2:30 - 4:00 PM  
- **Topics**:  
Homology and sequence alignments, Dynamic programming, Sequence similarity and database searching, Interpreting search results – E value, Needleman and Wunsch, Smith and Waterman, The BLAST algorithm.

#### Lab (1-2): 	Sequence alignment algorithms and applications  
- **Instructor**: 	Dr. Barry Grant  
- **Time**: 		2:30 – 4:00 PM, Jan 21 (Thursday) or Jan 22, 10:30 - 12:00 PM, (Friday)  
- **Topics**:  
Aligning novel sequences with previously characterized genes or proteins provides important insights into their common attributes and evolutionary origins. In this lab session we will explore the principles underlying the computational tools that can be used to compute and evaluate sequence alignments.  
- **Material**: [Lab worksheet]

<br>

#### Lecture (1-3): **Structural bioinformatics**  
- **Instructor**: 	Dr. Barry Grant  
- **Time**: 		Jan 26 (Tuesday), 2:30 - 4:00 PM  
- **Topics**:  
Major goals, current research challenges, and application areas of structural bioinformatics. Key concepts covered include: Sequence-structure-function relationships; Energy landscapes; Physics and knowledge based modeling approaches for describing the structure, energetics and dynamics of biomolecules computationally. 

#### Lab (1-3): 	Structure visualization, analysis and small molecule docking  
- **Instructor**: 	Dr. Barry Grant  
- **Time**: 2:30 – 4:00 PM, Jan 28 (Thursday) or Jan 29, 10:30 - 12:00 PM, (Friday)  
- **Topics**:  
The RCSB PDB resource.  Interactive biomolecular structure visualization with VMD.  Structural analysis of protein families and prediction of protein flexibility with Bio3D.  In silico docking and virtual screening strategies for drug design.
- **Material**: [Lab worksheet]

<br>

#### Lecture (1-4): **Genome informatics**  
- **Instructor**: 	Dr. Ryan Mills  
- **Time**: 		Feb 2 (Tuesday), 2:30 - 4:00 PM  
- **Topics**:  
High throughput sequencing technologies, biological applications and bioinformatics analysis methods.  Web interfaces to genome databases at Ensembl and UCSC.  


#### Lab (1-4): 	Web resources for analyzing genomic data 
- **Instructor**: 	Drs. Ryan Mills and Barry Grant
- **Time**: 		2:30 – 4:00 PM, Feb 4 (Thursday) or Feb 5, 10:30 - 12:00 PM, (Friday)
- **Topics**:
The Galaxy web-based platform for genomic data analysis. Manipulation and analysis of next generation sequencing data sets.
- **Material**: [Lab worksheet]


<br>

### Reference material
[Class Handout](../class-material/handout_day2-2_python.pdf)
<!--- files dont exist yet...
[Slides-2.1]()
[Slides-2.2]()
-->

